tensorflow/python/util/dispatch.py

```diff
@@ -43,6 +45,10 @@
 _GLOBAL_DISPATCHERS = []
 
 
+# Define type variables
```

Comment:
Let's make this comment more informative - this is a type var for decorators (or omit the comment altogether).